### ISSUE TITLE
can not wol over internet

### ISSUE THREAD

**Issue by Khajrur** [basildane/WakeOnLAN#81]
can not WOL or Shutdown over Internet. I recheck MAC Address, port, ip & UID & password is write
but i can not wake on lane or can not shutting Down. when i try to shut down the "error: The RPC server is unavailable. (Exception from HRESULT: 0x800706BA" came up.

**Comment by Khajrur** [comment]
can not WOL or Shutdown over Internet. I recheck MAC Address, port, ip & UID & password is write
but i can not wake on lane or can not shutting Down. when i try to shut down the "error: The RPC server is unavailable. (Exception from HRESULT: 0x800706BA" came up.

**Comment by basildane** [comment]
Yes, you cannot send RPC traffic over the Internet.  You would have to use a VPN or some way to connect the remote devices.

As for WOL, those are "broadcast" messages and they too cannot be sent over the Internet without special handling.  I have a program called "agent" which can run on the remote network, listens for incoming wol messages, and broadcasts them on your LAN.  Router and firewall must be setup to support this.
